### PR TITLE
Changes to story-postcard.hbs, date placement with clock icon

### PR DESCRIPTION
**Pull request by Eykf1997**
Changing of date placement

### CODE REVIEW

**Review by wei2912 — CHANGES_REQUESTED** [review]
Code looks good, although I think there's a leftover " - " in the post title and excessive space from excerpt to date. Also, the date's a bit too close to the corner of the card.

image

**Comment by Eykf1997** [comment]
Updated date placement.

File: partials/story-postcard.hbs [hunk 1]
```diff
@@ -5,18 +5,22 @@
         {{else}} {{!-- Placeholder --}}
         <img class="object-cover m-0 hidden md:block md:min-w-[40%] md:max-w-[40%] md:h-full lg:w-full lg:max-w-full lg:h-60" src="https://via.placeholder.com/100/FFFFFF/FFFFFF" alt="{{title}}" />
         {{/if}}
-        <div class="flex-grow bg-gray-100 px-3 pt-2 pb-4">
+        <div class="flex-grow bg-gray-100 px-3 pt-3 pb-5 relative">
             <div>
                 <p class="text-sm my-1">
                     {{#if primary_tag}}
                     {{primary_tag.name}} -
```
Comment (wei2912) [inline partials/story-postcard.hbs:12]: Do delete the leftover " - " here.

File: partials/story-postcard.hbs [hunk 2]
```diff
@@ -5,18 +5,22 @@
...
         <img class="object-cover m-0 hidden md:block md:min-w-[40%] md:max-w-[40%] md:h-full lg:w-full lg:max-w-full lg:h-60" src="https://via.placeholder.com/100/FFFFFF/FFFFFF" alt="{{title}}" />
         {{/if}}
-        <div class="flex-grow bg-gray-100 px-3 pt-2 pb-4">
+        <div class="flex-grow bg-gray-100 px-3 pt-3 pb-5 relative">
             <div>
                 <p class="text-sm my-1">
                     {{#if primary_tag}}
                     {{primary_tag.name}} - 
                     {{else}}
                     No tag -
```
Comment (wei2912) [inline partials/story-postcard.hbs:14]: See L12.

File: partials/story-postcard.hbs [hunk 3]
```diff
@@ -5,18 +5,22 @@
...
                     {{#if primary_tag}}
                     {{primary_tag.name}} - 
                     {{else}}
                     No tag - 
                     {{/if}}
-                    <time datetime="{{date format="YYYY-MM-DD"}}">{{date published_at format="D MMM YYYY"}}</time>
                 </p>
                 <p class="text-lg font-bold my-2">{{title}}</p>
                 <p class="text-sm my-1 max-h-20 line-clamp-4">{{excerpt}}</p>
+
```
Comment (wei2912) [inline partials/story-postcard.hbs:19]: Please delete this blank line.

File: partials/story-postcard.hbs [hunk 4]
```diff
@@ -5,18 +5,22 @@
...
+            </div>
+            <div class="text-sm absolute right-4 bottom-4" >
```
Comment (wei2912) [inline partials/story-postcard.hbs:21]: Delete the extra space before the `>`

**Review by wei2912 — CHANGES_REQUESTED** [review]
Few minor changes to be made, otherwise LGTM.

**Comment by wei2912** [comment]
Squashed and merged in https://github.com/AdvisorySG/ghost-advisory-theme/commit/65314182c5ad6926d7dba7f00b19ac8900bcb613, thanks!

**Pull request closed without merging**